composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/security-csrf": "^3.4 || ^4.0",
         "symfony/translation": "^3.4 || ^4.0",
         "symfony/validator": "^3.4 || ^4.0",
-        "twig/twig": "^2.0"
+        "twig/twig": "^2.9"
```

Comment:
`^2.0` already allows 2.9.0, so this does nothing, right? :thinking:

Reply:
It disallow 2.8 and lower, do we need this?

Reply:
as said in a different pr 2.9 was released 3 days ago, I don't think we want this